tests/test.sh

```
@@ -170,39 +185,42 @@ do
       continue
    fi
 
-   # Redirection to dev/null apparently needed for CP2K tests.
-   # Otherwise, STDIN is screwed up. I have no idea why.
-   # http://stackoverflow.com/questions/1304600/read-error-0-resource-temporarily-unavailable
-   # TODO: Figure out a different solution
+   # For special cases such as REMD, we need a more complicated
+   # test setup. If a file, test.sh is present in the test directory,
+   # we will use it.
    if [[ -f "test.sh" ]];then
 
+      # Redirection to dev/null apparently needed for CP2K tests.
+      # Otherwise, STDIN is screwed up. I have no idea why.
+      # http://stackoverflow.com/questions/1304600/read-error-0-resource-temporarily-unavailable
+      # TODO: Figure out a different solution
       #./test.sh $ABINEXE 2> /dev/null
       ./test.sh $ABINEXE || true
 
    else
       if [[ -f "velocities.in" ]];then
-         $ABINEXE -v "velocities.in" > output || true
+         $ABINEXE -v "velocities.in" > $ABINOUT || true
       else
-         $ABINEXE > output || true
+         $ABINEXE > $ABINOUT || true
```

Since we're using the `set -e` BASH settings, we need this syntax `command || true` for commands that are allowed to fail. ABIN needs to be allowed to fail since we purposefully let it fail in some tests.